src/aat/resources/features/F-040/F-040.feature

```diff
@@ -0,0 +1,21 @@
+@F-040
+Feature: F-040: Get Case for Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-576
+  Scenario: must return successfully all and only the respective fields with READ access for Solicitors
+    Given a user with [an active Solicitor profile in CCD with a specific variation of ACLs on a case type]
+    And a user with [another active solicitor profile with different variation of ACLs on the same case type]
```

Comment:
should we also mention the privileged user (solicitor 2) here?

Reply:
It is actually Solicitor3 here. Updated the code to include solicitor 3 keyword.

Reply:
I mean in addition to solicitor 1 and solicitor 3, we should also mention solicitor 2 who is making calls in this scenario

Reply:
similar to this: https://github.com/hmcts/ccd-data-store-api/pull/738/files#diff-87a421b969e48e84889cb88e66334294R72-R74

Reply:
Updated the file @bmcts .